saleor/plugins/webhook/tasks.py

```diff
@@ -43,13 +52,31 @@ def trigger_webhooks_for_event(event_type, data):
     webhooks = webhooks.select_related("app").prefetch_related(
         "app__permissions__content_type"
     )
+    return webhooks
+
 
+@app.task
+def trigger_webhooks_for_event(event_type, data):
+    """Send a webhook request for an event as an async task."""
+    webhooks = _get_webhooks_for_event(event_type)
     for webhook in webhooks:
         send_webhook_request.delay(
             webhook.pk, webhook.target_url, webhook.secret_key, event_type, data
         )
 
 
+def trigger_webhook_sync(event_type: str, data: str, app: "App"):
+    """Send a synchronous webhook request."""
+    webhooks = _get_webhooks_for_event(event_type, app.webhooks.all())
```

Comment:
Should we filter out not active webhooks?

Reply:
`_get_webhooks_for_event` filters webhooks by `is_active=True` internally.